src/styles/blocks/product-cards.scss

```diff
@@ -0,0 +1,124 @@
+.main {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 46px 48px;
+
+  padding: 50px 40px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 1fr);
+  }
+
+  @media (min-width: 1200px) {
+    grid-template-columns: repeat(4, 1fr);
+    padding: 50px 128px;
+  }
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: $color_white;
+  border-radius: 5px;
+  border: 1px solid $hover_color;
+
+  transition-property: transform;
+  transition-duration: 300ms;
+
+  &__photo {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
```

Comment:
You alredy reset your default style so you don't need to write 0 margins to top left right directions. Would be better to write `margin bottom`